### PR TITLE
[Sorry] Update screenshots, Software Manual, Setup page

### PR DESCRIPTION
**Pull request by gilgongo**
# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

Re-arranges Software Manual to cover new features as of 3.8.0 and removing some screenshots to lower maintenance overhead in future. Minor changes to text for style, added TOC. 

### Angst

I realise of course that I should have done this **BEFORE** we created the translation branch. Not sure what to do ...

### CODE REVIEW

File: wiki/en/en-Software-Manual.md [hunk 1]
```diff
@@ -4,104 +4,95 @@ title: "Software Manual"
 lang: "en"
 permalink: "/wiki/Software-Manual"
 ---
-# Jamulus Software Manual
+# Jamulus User Manual
+{:.no_toc}
 
-## Main Window
+This manual documents the Jamulus client appliction for use by musicians and singers using the software to connect to a server. 
+If you wish to set up your own server, see the [Server Administrators' Manual](Server-Admin-Manual)
```
Comment (gilgongo) [inline wiki/en/en-Software-Manual.md]:
```suggestion
```

File: wiki/en/en-Server-Admin-Manual.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: wiki
+title: "Server Adminstrator's Manual"
+lang: "en"
+permalink: "/wiki/Server-Admin-Manual"
+---
+
+# Jamulus Server Adminstrator's Manual
```
Comment (jujudusud) [inline wiki/en/en-Server-Admin-Manual.md]: Administrator's instead of Adminstrator's

File: wiki/en/en-Software-Manual.md [hunk 3]
```diff
@@ -4,104 +4,94 @@ title: "Software Manual"
 lang: "en"
 permalink: "/wiki/Software-Manual"
 ---
-# Jamulus Software Manual
+# Jamulus User Manual
+{:.no_toc}
 
-## Main Window
+This manual documents the Jamulus client appliction for use by musicians and singers using the software to connect to a server.
```
Comment (jujudusud) [inline wiki/en/en-Software-Manual.md]: application ->appliction

**Review by jujudusud — CHANGES_REQUESTED** [review]
two letters missing ...

**Comment by jujudusud** [comment]
Please Merge in changes, 
after that we will contact all the translators showing what as been changed. ;-)

**Comment by ignotus666** [comment]
If you want to push this to the translate branch to get it in before the next release it's ok by me.

**Comment by jujudusud** [comment]
> If you want to push this to the translate branch to get it in before the next release it's ok by me.

To the "changes" branch ?

File: wiki/en/en-Server-Admin-Manual.md (hunk 2 above)
Comment (gilgongo) [reply wiki/en/en-Server-Admin-Manual.md]: I think it might also need to be "Administrator" not "Administrator's", since there will also be "User Manual".

**Review by jujudusud — APPROVED** [review]
It's ok for me. Ready to be merged.

**Comment by ignotus666** [comment]
> To the "changes" branch ?

No, to 'translate'. If it goes into 'changes' I presume it won't get translated or go live until the release after this one, or at least some time after this release. What I meant was that it's ok by me if it goes into the current 'translate' branch so it can all go live for this release, but it doesn't only depend on me - the other translators would have to weigh in as it kind of "breaks" the translation protocol.

**Comment by gilgongo** [comment]
We've merged them to changes in the hope that a remedial translation can be merged to translate from there. @jujudusud says he can undertake the work...

**Comment by ignotus666** [comment]
Ok, so if I want to update the ES translation I have to do it on changes then?

**Comment by gilgongo** [comment]
To quote @jujudusud: "to translate we just have to go to changes branch, copy past the en file locally (or just read it in raw mode in GH interface) and translate on translate3_8_0 branch." We're talking about 3 files BTW.

**Comment by jujudusud** [comment]
> Ok, so if I want to update the ES translation I have to do it on changes then?

The modifications in english are made to "changes" branch.
The translations of all languages are made in "translate3_8_0" branch.

So, you have to refer to a file in english using something like this place to have the latest english file : [https://github.com/jamulussoftware/jamuluswebsite/blob/changes/wiki/en/en-Software-Manual.md](url)
or the raw file if you want : [https://raw.githubusercontent.com/jamulussoftware/jamuluswebsite/changes/wiki/en/en-Software-Manual.md](url)
and translate on your own fork, on your own translate3_8_0 branch, on your own language : [https://github.com/ignotus666/jamuluswebsite/blob/translate3_8_0/wiki/es/es-Software-Manual.md](url)

the files to be retranslated are :
- en-Software-Manual.md
- en-Setup.md 
- en-Server-Admin-Manual.md
- en-Installation-for-Linux.md 

after translating, open a new PR and merge in translate3_8_0 branch

Is it ok for you @ignotus666 ?

**Comment by ignotus666** [comment]
Done.

**Pull request merged**